logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/user/SetUserHandleUseCaseTest.kt

```diff
@@ -45,6 +49,15 @@ class SetUserHandleUseCaseTest {
         given(userRepository)
             .coroutine { updateSelfHandle(handle) }
             .then { Either.Right(Unit) }
+        given(userRepository)
+            .coroutine { updateLocalSelfUserHandle(handle) }
+            .then { }
```

Comment:
```suggestion```

You can delete stubs of `Unit` functions if you define the Mock like this:

```kotlin
    @Mock
    private val userRepository = configure(mock(classOf<UserRepository>())) { stubsUnitByDefault = true }
```

Reply:
Nice one, thanks! I changed it 🚀